src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfile/AntigenTestProfileViewController.swift

```diff
@@ -11,11 +11,15 @@ class AntigenTestProfileViewController: UIViewController, UITableViewDataSource,
 	init(
 		store: AntigenTestProfileStoring,
 		didTapContinue: @escaping (@escaping (Bool) -> Void) -> Void,
+		didTapInfoProfile: @escaping () -> Void,
```

Comment:
```suggestion
		didTapProfileInfo: @escaping () -> Void,
```

Just a tiny improvement

Reply:
Will change this on a following PR. So this one can be merge immediately.